OpenRA.Mods.Common/Scripting/Properties/PlayerExperienceProperties.cs

```diff
@@ -0,0 +1,43 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2016 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using Eluant;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Scripting;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Scripting
+{
+	[ScriptPropertyGroup("Player")]
+	public class PlayerExperienceProperties : ScriptPlayerProperties, Requires<PlayerExperienceInfo>
```

Comment:
`Requires<PlayerExperienceInfo>`! This breaks all scripts for me.

We can either wait for #11627 to add the trait to the default mod rules, or take something like https://github.com/OpenRA/OpenRA/compare/bleed...abcdefg30:reqExp.